i3pystatus/mail/ews.py

```diff
@@ -0,0 +1,74 @@
+import exchangelib
+import contextlib
+import time
+
+from i3pystatus.mail import Backend
+
+
+EWS_EXCEPTIONS = (Exception)
```

Comment:
Is this necessary considering there is only one exception?

Reply:
Note that EWS_EXCEPTIONS is not a tuple here, because `(1) == 1` in Python.